src/components/SipProvider/index.js

```diff
@@ -287,9 +287,7 @@ export default class SipProvider extends React.Component {
           return;
         }
 
-        this.remoteAudio.src = window.URL.createObjectURL(
-          rtcSession.connection.getRemoteStreams()[0],
-        );
+        this.remoteAudio.srcObject = rtcSession.connection.getRemoteStreams()[0];
```

Comment:
is it guaranteed not to be null? `(rtcSession.connection.getRemoteStreams() || [])[0]` can be used otherwise